src/component/AssetProfile.tsx

```diff
@@ -0,0 +1,74 @@
+import { StrategicAssets } from '../data/strategicAssets.ts'
+import useDataslateStore from '../stores/dataslateStore.ts'
+
+interface Props {
+  asset: StrategicAssets
+  buyMode?: boolean
+}
+
+const StrategicAssetsProfile = (props: Props) => {
+  const { asset, buyMode } = props
+  const { name, description, rule } = asset
+  const selectedstrategicAssets = useDataslateStore(
+    (state) => state.selectedDataslate?.baseOfOperations.strategicAssets,
+  )
+
+  const selectedstrategicAssetsLength =
+    useDataslateStore(
+      (state) =>
+        state.selectedDataslate?.baseOfOperations.strategicAssets.length,
+    ) ?? 0
+
+  const addtoStrategicAssets = useDataslateStore(
+    (state) => state.addtoStrategicAssets,
+  )
+
+  const removeFromStrategicAssets = useDataslateStore(
+    (state) => state.removeFromStrategicAssets,
+  )
+
+  const assetCapacity = useDataslateStore(
+    (state) => state.selectedDataslate?.baseOfOperations.assetCapacity || 0,
+  )
+
+  const isNameInBase = selectedstrategicAssets?.some(
+    (selectedstrategicAssets) => selectedstrategicAssets.name === asset.name,
+  )
+  const isAssetSelectable = () => {
+    if (assetCapacity > selectedstrategicAssetsLength && !isNameInBase) {
```

Comment:
this can be simplified to just a const


`const isAssetSelectable = assetCapacity > selectedstrategicAssetsLength && !isNameInBase`